nexus/social_media/tasks.py

```diff
@@ -0,0 +1,23 @@
+# Standard Library
+from datetime import datetime
+
+# Nexus Stuff
+from nexus.social_media.models import Post
+from nexus.social_media.services import post_to_facebook
+from nexus.celery import app
+from settings.common import MAX_POSTS_AT_ONCE
+# from nexus.social_media.services import post_to_twitter
+
+
+@app.task(name='queue_posts')
+def queue_posts():
+    posts = Post.objects.filter(
```

Comment:
I think all this can be put in a service and just calling that service in `queue_posts`.

@CuriousLearner what could be a probable place for this code?

Reply:
`services.py` should be good.